weatherman.py

```diff
@@ -0,0 +1,246 @@
+__weather__ = {}  # for storing weather records
```

Comment:
Declaring global variable is a bad practice. Rethink on your approach of storing and consuming weather records between functions.

Reply:
Any particular reason for using prefix and trailing underscore convention?